tests/test_extract_chicago_ridership.py

```diff
@@ -0,0 +1,37 @@
+import pytest
+import datetime
+from ingestion.extract_chi_ridership_data import extract_daily_data, DATASETS, CHI_TZ
+
+#########################
+# Extract daily data tests
+#########################
+
+
+@pytest.mark.parametrize(
+    "url,date,expected",
+    [
+        (
+            DATASETS["BUS_RIDERSHIP"]["URL"],
+            datetime.datetime(2023, 2, 28, tzinfo=CHI_TZ),
+            125,
+        ),
+        (
+            DATASETS["BUS_RIDERSHIP"]["URL"],
+            datetime.datetime(2023, 3, 4, tzinfo=CHI_TZ),
+            94,
+        ),
+        (
+            DATASETS["SUBWAY_RIDERSHIP"]["URL"],
+            datetime.datetime(2023, 2, 28, tzinfo=CHI_TZ),
+            143,
+        ),
+        (
+            DATASETS["SUBWAY_RIDERSHIP"]["URL"],
+            datetime.datetime(2023, 3, 4, tzinfo=CHI_TZ),
+            143,
+        ),
+    ],
+)
+def test_extract_daily_data(url, date, expected):
```

Comment:
I've taken a look at the data extraction and ingestion scripts and they generally look good. For the tests section, should we add a test to check that we're not adding duplicate values into the db?